pkgs/development/python-modules/nupd/default.nix

```diff
@@ -0,0 +1,74 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  aiohttp,
+  attrs,
+  frozendict,
+  inject,
+  loguru,
+  nbdb,
+  nix-prefetch-git,
+  nurl,
+  platformdirs,
+  typer,
+
+  # tests
+  pytestCheckHook,
+  aioresponses,
+  pytest-asyncio,
+  pytest-cov,
+  pytest-mock,
+}:
+buildPythonPackage rec {
+  pname = "nixpkgs-updaters-library";
+  version = "0.1.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.12";
+
+  src = fetchFromGitHub {
+    owner = "PerchunPak";
+    repo = "nixpkgs-updaters-library";
+    tag = "v${version}";
+    hash = "sha256-/hCtWTVasjWUSJzsBxbWNuBPGzaT0r8ukWL5g3G2q3Q=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    aiohttp
+    attrs
+    frozendict
+    inject
+    loguru
+    nbdb
+    nix-prefetch-git
```

Comment:
That is not a python module, is it?

Reply:
Oh, you are right. I hope `buildInputs` is the right attribute for it